### ISSUE TITLE
CoBlocks/Masonry block bunched / smushed together on AT sites

### ISSUE THREAD

**Issue by kylemcph** [Automattic/wp-calypso#36365]
#### Steps to reproduce
1. Start a WordPress.com, and a Business plan, and install any plugin to go ATomic
2. Make sure CoBlocks plugin is installed (should be automatically).
3. Create a post with the CoBlocks/Masonry block, add any number of images
4. Publish the post.

#### What I expected

Images to be displayed in a gallery, similar to what happens when with a simple site.

#### What happened instead

Images displayed smushed together.

#### Browser / OS version

MacOS 10.14.6

Same result on all browsers tested: 

1. Chrome Version 77.0.3865.90 (Official Build) (64-bit)
2. Opera Version:63.0.3368.94
3. Firefox 69.0.1 (64-bit)
4. Safari Version 13.0.1 (14608.2.11.1.11)

## Simple Site:

![](https://cld.wthms.co/7ncVXI+)
Direct link to image: https://cld.wthms.co/7ncVXI

--- 

## Atomic Site: 

![](https://cld.wthms.co/d56qnb+)
Direct link to image: https://cld.wthms.co/d56qnb

#### Context / Source

I tested this with various different themes and block settings, and the result was always the same. Interestingly enough, each time I refresh the browser something different appears. Sometimes random images are displayed correctly, other times they are all smushed. 

Originally pointed out by a user #15507304-hc

**Comment by annbingle** [comment]
1. The user is following-up on this reported issue earlier about Masonry block.
2. Checked on the earlier follow-up ticket but it has been closed though the GH issue isn't fixed yet. https://github.com/Automattic/wp-calypso/issues/36365
3. Old follow-up ticket: https://woothemes.zendesk.com/agent/tickets/2380160
4. I'm not sure if this is tied up to the Masonry block issue they reported but they said that Calypso isn't working at all for them on all their devices, Desktop, laptop, tablet and phone. Only WP Admin works for now.
5. Whenever they try to use Calypso, it just freezes the whole window.
6. Checked PHP-Errors in CLI and I got below:
[28-Oct-2019 20:18:08 UTC] PHP Notice:  Undefined index: external_display in /wordpress/plugins/jetpack/7.8/modules/publicize/publicize.php on line 371
